setup.py

```diff
@@ -282,14 +282,22 @@ def finalize_options(self):
         'six >= 1.5.0',
         'smart_open >= 1.2.1',
     ],
+    tests_require=[
+        'testfixtures',
+        'unittest2',
+        'Morfessor==2.0.2a4',
+        'scikit-learn',
+        'pyemd'
+    ],
     extras_require={
         'distributed': ['Pyro4 >= 4.27'],
         'wmd': ['pyemd >= 0.2.0'],
         'test': [
```

Comment:
It's better that I remove the duplication of the list of libraries (use common variable) because the option `[test]` is very convenient for some cases.